### DISCUSSION TITLE
Ruby syntax validator, is this available

### DISCUSSION THREAD

**Discussion by RiteshAryal** [ajaxorg/ace#4592] [Q&A] [answered]
Hi, 
I went through this page https://github.com/ajaxorg/ace/wiki/Syntax-validation and looking like there no validator for Ruby code. 

My question is - is it on the way which I can apply straight away OR do you want to suggest anything if I want to apply Syntax validation on the Ruby script.

Please suggest.

**Comment by andrewnester** [comment, marked as answer]
there is no default Ruby syntax validation support at the moment, you can add custom validation yourself as described here
https://github.com/ajaxorg/ace/wiki/Syntax-validation

**Reply by RiteshAryal** [reply]
Thanks for your suggestion.
